### DISCUSSION TITLE
Facing issue in webpack - when an npm module which internally uses dynamic imports

### DISCUSSION THREAD

**Discussion by klynsaha** [webpack/webpack#17082] [Q&A]
I am using an npm module that internally uses dynamic imports. However, my project's webpack configuration is not properly bundling the dependency. Specifically, it is attempting to retrieve the npm module's dynamic import file from the root of my domain.

Could someone please help me resolve this issue or direct me to a useful resource?

**Comment by alexander-akait** [comment]
Please provide an example of code
